src/core/config.py

```diff
@@ -1,15 +1,14 @@
 import os
-import google.generativeai as Client
 from github import Github
-from typing import Tuple
 
 class Config:
```

Comment:
This line is importing `google.generativeai` but it's not used anywhere in the code.  Please remove this import.